content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -12,7 +12,36 @@ match:
     repositories
   ]
 tooltip: >-
-  Initialized by running `dvc init` in the **workspace** (typically a Git
-  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  A DVC project is a directory initialized by `dvc init` that contains a `.dvc`
+  directory. It contains all <abbr>metafiles</abbr> related to project and also
+  includes the underlying <abbr>cache</abbr>.
+  [📖](/doc/user-guide/basic-concepts/dvc-project)
```

Comment:
I think we should avoid changing tooltips in these PRs unless there's a strong reason. Tooltips are currently written to be as short as possible and checked visually. Also, in this case the change states that a project is a directory when that's not necessarily the case conceptually.

Reply:
Adding the 📖 links to tooltips in these PRs is great though.